.github/workflows/deploy-mcp-auth-stage.yml

```
@@ -0,0 +1,50 @@
+name: Deploy MCP Auth to DigitalOcean Stage
+
+on:
+  workflow_run:
+    workflows: ["Build and Publish Gauzy MCP Auth Image Stage"]
+    branches: [stage]
+    types:
+      - completed
+
+jobs:
+  deploy-mcp-auth-stage:
+    runs-on: buildjet-4vcpu-ubuntu-2204
```

actionlint (1.7.8)

label "buildjet-4vcpu-ubuntu-2204" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)